tests/Integrations/Slim/V4/CommonScenariosTest.php

```diff
@@ -0,0 +1,119 @@
+    {
+        $traces = $this->tracesFromWebRequest(function () use ($spec) {
+            $this->call($spec);
+        });
+
+        $this->assertFlameGraph($traces, $spanExpectations);
+    }
+
+    public function provideSpecs()
+    {
+        return $this->buildDataProvider(
+            [
+                'A simple GET request returning a string' => [
+                    SpanAssertion::build(
+                        'web.request',
+                        'slim_test_app',
+                        'web',
+                        'GET /simple'
+                    )->withExactTags([
+                        'slim.route' => 'Closure::__invoke',
+                        'http.method' => 'GET',
+                        'http.url' => 'http://localhost:9999/simple',
+                        'http.status_code' => '200',
+                    ])->withChildren([
+                        SpanAssertion::build(
+                            'slim.route',
+                            'slim_test_app',
+                            'web',
+                            'Closure::__invoke'
+                        )->withExactTags([
+                            'slim.route.name' => 'simple-route',
+                        ])
+                    ]),
+                ],
+                'A simple GET request with a view' => [
+                    SpanAssertion::build(
+                        'web.request',
+                        'slim_test_app',
+                        'web',
+                        'GET /simple_view'
+                    )->withExactTags([
+                        'slim.route' => 'Closure::__invoke',
+                        'http.method' => 'GET',
+                        'http.url' => 'http://localhost:9999/simple_view',
+                        'http.status_code' => '200',
+                    ])->withChildren([
+                        SpanAssertion::build(
+                            'slim.route',
+                            'slim_test_app',
+                            'web',
+                            'Closure::__invoke'
+                        )->withChildren([
+                            SpanAssertion::build(
+                                'slim.view',
+                                'slim_test_app',
+                                'web',
+                                'simple_view.phtml'
+                            )->withExactTags([
+                                'slim.view' => 'simple_view.phtml',
+                            ])
+                        ])
+                    ]),
+                ],
+                'A GET request with an exception' => [
+                    SpanAssertion::build(
+                        'web.request',
+                        'slim_test_app',
+                        'web',
+                        'GET /error'
+                    )->withExactTags([
+                        'slim.route' => 'Closure::__invoke',
```

Comment:
We are still attaching `slim.route` to the root span; did we mean to do that? I'm not sure why we'd want it on the root span as well as having it as a span resource. I'm thinking that wherever we stick the handler name we should put the route name too, since they kind of go together. So either add `slim.route.name` to the root as well or remove `slim.route` from the root, I think?

Reply:
Good call. I added it to the root span as `slim.route.name`.

Reply:
Thanks. Do you think we should rename the meta on root span `span.route` -> `span.route.handler`? We have `slim.route` and `slim.route.name` now. In the Slim 3 integration we used `slim.route.controller`.

Reply:
Makes sense to me! Added `slim.route.handler` as meta to the root span.